chalice/utils.py

```diff
@@ -101,6 +116,84 @@ def set_file_contents(self, filename, contents, binary=True):
         with open(filename, mode) as f:
             f.write(contents)
 
+    def unpack_zipfile(self, zipfile_path, unpack_dir):
```

Comment:
This is picky, but I think we should use verbs that are commonly associated with whatever the noun is, so for a zipfile, I would expect "unzip", or "extract" but not "unpack".